www/components/scroll/scroll.js

```diff
@@ -8,33 +8,40 @@ class scroll extends LitElement {
   connectedCallback() {
     super.connectedCallback();
     window.addEventListener('hashchange', this.handleHashChange.bind(this), false);
+    window.addEventListener('load', this.handleHashChange.bind(this), false);
   }
 
   disconnectedCallback() {
     window.removeEventListener('hashchange', this.handleHashChange.bind(this), false);
+    window.removeEventListener('load', this.handleHashChange.bind(this), false);
     super.disconnectedCallback();
   }
 
   handleHashChange() {
     // on hash change child event, scroll content
     const contains = (selector, text) => {
       const elements = this.querySelectorAll(selector);
+      // convert NodeList to Array
+      const elementsArray = Array.from(elements);
 
-      return Array.prototype.filter.call(elements, (element) =>{
+      return Array.prototype.filter.call(elementsArray, (element) => {
```

Comment:
Do we still think we can make this refactor possible?
https://github.com/ProjectEvergreen/greenwood/pull/246#discussion_r338226468